README.md

```diff
@@ -1 +1,35 @@
-# school-library
\ No newline at end of file
+# school-library
+
+> A school library app to record what books are in the library and who borrows them. Developed on Ruby.
+
+## Technologies used:
+
+- Ruby
+
+## Author
+
+👤 **Ghilain Ishimwe** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- [GitHub](https://github.com/Ghilain)
+- [Twitter](https://twitter.com/GhilainIshimwe)
+- [LinkedIn](https://www.linkedin.com/in/ghilain-ishimwe/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
```

Comment:
- [ ] Kindly add the `Getting Started` section of your Readme as seen in the Microverse template and add the information needed to get your program up and running.

Reply:
Thanks @julie-ify